t/plugin/kafka-logger2.t

```diff
@@ -340,6 +340,50 @@ qr/not found topic, retryable: true, topic: undefined_topic, partition_id: -1/
                         key= "key1",
                     },
                 },
+                {
+                    input = {
+                        brokers = {
+                        },
+                        kafka_topic = "test",
```

Comment:
Add another test case that the `host` or the `port` field is missing.

Reply:
Fixed.